### PR TITLE
docs: Add scripts for running on kubernetes on gce and aws

### PR DESCRIPTION
**Pull request by a-robinson**
This is actually really simple, because there's an automatic volume
provisioner on GCE and AWS. It's only an alpha feature, but so is
petsets, so using it doesn't limit us any more than we already are.

@jseldess @tschottdorf

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/9515)

### CODE REVIEW

**Comment by tschottdorf** [comment]
> High-performance use cases on a private Kubernetes cluster should consider a DaemonSet deployment.

Remove this (unless you think this is really good advice)

The document makes no (?) mention of the fact that this is only available on alpha kubernetes. Consider pointing that out more clearly and also add a reference that explains how to set up an alpha cluster.

**Comment by tschottdorf** [comment]
LGTM otherwise.

**Comment by a-robinson** [comment]
Done.

**Review by tbg — APPROVED** [review]
No link to tutorial for setting up alpha cluster on the/and?

**Comment by tschottdorf** [comment]
*gce/aws?

File: cloud/kubernetes/README.md [hunk 1]
```diff
@@ -47,7 +53,8 @@ which will lead to a lot of trouble.
 The deployment is written for a use case in which dynamic provisioning is
 available. When that is not the case, the persistent volume claims need
 to be created manually. See [minikube.sh](minikube.sh) for the necessary
-steps.
+steps. If you're on GCE or AWS, where dynamic provisioning is supported, no
+manual work is needed to create the persistent volumes.
 
 ## Testing locally on minikube
```
Comment (jseldess) [inline cloud/kubernetes/README.md]: Should we add another section to the README for aws/gce deployment?

**Review by jseldess — COMMENTED** [review]
Thanks for adding the gce and aws scripts, @a-robinson! I'll update the docs soon as well.

File: cloud/kubernetes/README.md (hunk 1 above)
Comment (a-robinson) [inline cloud/kubernetes/README.md]: Yeah, done.

**Comment by a-robinson** [comment]
Whoops, added the instructions for creating kubernetes clusters on aws/gce.

**Comment by a-robinson** [comment]
Updated to mention that GKE now supports alpha features in specially created clusters.

**Comment by tschottdorf** [comment]
Reviewed 2 of 3 files at r1, 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks pending.

---

*[cloud/kubernetes/README.md, line 84 at r3](https://reviewable.io:443/reviews/cockroachdb/cockroach/9515#-KSmOFiNGq5jdtM1tnup:-KSmOFiNGq5jdtM1tnuq:b5r14n9) ([raw file](https://github.com/cockroachdb/cockroach/blob/989be7bb642f9009fb22da2be4e11c6dc5f77d33/cloud/kubernetes/README.md#L84)):*
> ```Markdown
> 
> ```shell
> gcloud alpha container clusters create NAME --enable-kubernetes-alpha
> ```

Oh, neat!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/9515)*

**Pull request merged**